aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -1324,7 +1324,7 @@ public void deliveryComplete(IMqttDeliveryToken token) {
      *
      * @return true equals ready to publish, false equals offline.
      */
-    boolean isReadyToPublish() {
+    public boolean isReadyToPublish() {
```

Comment:
Personally, I tend to wait to change/make things public until necessary.  However, this seems like something that might be useful to encourage use.